addons/website_form/controllers/main.py

```diff
@@ -100,6 +100,9 @@ def integer(self, field_label, field_input):
     def floating(self, field_label, field_input):
         return float(field_input)
 
+    def html(self, field_label, field_input):
```

Comment:
In website forms, for both html/text type of fields, we provide simple textarea to users for entering the data. And, we store the data as it is for both of them.

But with this change, we're trying to convert plain text (entered by user in the online form for html type of fields) into to html, so for example it can replace `\n` with `<br>` and properly formatted data can be saved. It's not part of the spec, just an improvement. We can remove it if you want.

Regarding the method name, it's an inner method doing the conversion, and we tried to match the rest of the method names based on fields types (like integer, one2many, many2many, binary...). If this change is to be kept, kindly suggest the method name.

Reply:
Seems ok. Could be great to add this explanation somewhere :)

Reply:
I have added the explanation in commit message. Will it be ok there?